mindmap/mindmap.py

```diff
@@ -634,7 +655,7 @@ def validate_score(points: int, weight: int) -> None:
 
         if weight:
             try:
-                weight = float(weight)
+                weight = int(weight)
```

Comment:
shouldn't this be a float?

Reply:
I don't think so. The submissions API doesn't allow decimal values for scores, so when calculating the weighted score we should avoid decimal values. Unless you think of a use case we're missing when using integers.